src/util/read-package-json.js

```diff
@@ -0,0 +1,7 @@
+import fs from 'fs-promise';
+import path from 'path';
+
+export default (dir) => {
+  const packageData = fs.readFileSync(path.resolve(dir, 'package.json'), 'utf8');
```

Comment:
It should work / look something like this one

https://github.com/electron-userland/electron-forge/blob/master/src/util/resolve-dir.js#L7